README.md

```diff
@@ -3,60 +3,70 @@
 
-The `galasaVersion` value is the version of Galasa you want to run with.  You should not use latest to ensure each pod in the Ecosystem is running at the same level.
+The `--wait` flag ensures the chart installation has completed before marking it as "Deployed". During the installation, the API pod waits for the etcd and RAS pods to initialise while the engine-controller, metrics, and resource-monitor pods wait for the API pod to initialise.
 
-The `externalHostname` value is the DNS hostname or IP address of the Kubernetes node that be used to access the Galasa NodePort services.
+The `galasaVersion` value is the version of Galasa you want to run. You should not use latest to ensure each pod in the Ecosystem is running at the same level.
 
-The Galasa Ecosystem Helm chart will deploy three persistent volumes.  If you need to provide a Kubernetes storage class for these PVs, then please use the following command:-
+The `externalHostname` value is the DNS hostname or IP address of the Kubernetes node that be used to access the Galasa NodePort services. If you are deploying to minikube, this can be retrieved by running `minikube ip`.
 
-    helm install --set storageClass=mysgtorageclass --set galasaVersion=0.23.0 --set externalHostname=my.host.name <release-name> galasa/ecosystem --wait 
+After the Ecosystem has been successfully deployed, you can run the following command to ensure the Ecosystem can be accessed externally to Kubernetes and a simple test engine can be run:
 
-After the Ecosystem has been successfully deployed, you can run the following command to ensure the Ecosystem can be accessed externally to Kubernetes and a simple test engine can be run:-
+```
+helm test <release-name>
+```
 
-    helm test <release-name>
+Once the Ecosystem is up and running, you will need to determine the URL of the Ecosystem bootstrap. Issue the command:
 
-Once the Ecosystem is up and running,  you will need to determine the URL of the Ecosystem bootstrap.  Issue the command:-
+```
+kubectl get svc
+```
 
-    kubectl get svc
-    
-Look for the api-external service and the node port associated with the 8080 port.   Combine that with the external hostname you provided.  for example:-
+Look for the api-external service and the node port associated with the 8080 port.   Combine that with the external hostname you provided. For example:
 
 ```
 test-api-external                 NodePort    10.107.160.208   <none>        9010:31359/TCP,9011:31422/TCP,8080:30960/TCP   18s
 ```
 
-If the external hostname you provided was `example.com`,  the bootstrap URL will be `http://example.com:30960/boostrap`.  You will enter this into the Eclipse plugin preferences, or the galasactl command.
+If the external hostname you provided was `example.com`, the bootstrap URL will be `http://example.com:30960/boostrap`. You will enter this into the Eclipse plugin preferences, or the galasactl command.
```

Comment:
This seems out of place. At one point an administrator is installing the ecosystem. This part is about a developer pointing to that ecosystem. Should be in a separate section. Perhaps in the eclipse docs (also?).